ow_faults/msg/SystemFaults.msg

```diff
@@ -1,3 +1,13 @@
 # System Fault Status Mask
 Header header
-uint64 value
\ No newline at end of file
+uint64 value
+
+# System=1, 
+# ArmGoalError=2, 
+# ArmExecutionError=4,
+# TaskGoalError=8, 
+# CamGoalError=16, 
+# CamExecutionError=32, 
+# PtGoalError=64, 
+# PtExecutionError=128, 
+# LanderExecutionError = 256};
```

Comment:
Minor: "};" at end.  Clearly copied from C code :)

Reply:
oop yea forgot to delete that part! Thanks!